test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/LocalClusterHandle.java

```diff
@@ -93,4 +93,9 @@ public interface LocalClusterHandle extends ClusterHandle {
      * Returns an {@link InputStream} for the given node log.
      */
     InputStream getNodeLog(int index, LogType logType);
+
+    /**
+     * Writes current keystore settings to keystore file on each node.
+     */
+    void writeToKeystoreFile();
```

Comment:
I'd rename this to something more generic like `updateSecrets` and expand on the javadoc to reference mutablesettingprovider since it doesn't make sense to use this outside that context.

Reply:
Went with `updateStoredSecureSettings` let me know if that works.